ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js

```diff
@@ -70,4 +72,19 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
       throw new Error(e);
     }
   }
+
+  /**
+   * Deletes the session recording
+   * @param {SessionRecording} recording
+   */
+  @action
+  @loading
+  @confirm('questions.delete-confirm')
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess('notifications.delete-success')
+  async delete(recording) {
```

Comment:
Don't we normally have the delete route action at the list route level? In this case the `session-recordings` route?

Reply:
that's right, I updated